components/content-blocks/ComplexTable/styles.js

```diff
@@ -0,0 +1,56 @@
+import styled, { css } from "styled-components";
+
+export const TableWrapper = styled.div`
+  max-width: 100vw;
+  width: 100%;
+  overflow: scroll;
+`;
+
+export const Table = styled.table`
+  width: 100%;
+
+  ${({ $styleAs }) =>
+    $styleAs === "secondary"
+      ? css`
+          --ComplexTable-cell-bg: var(--neutral10);
+          --ComplexTable-border: 0;
+          border-spacing: 5px;
+        `
+      : css`
+          --ComplexTable-border: 1px solid var(--black);
+          --ComplexTable-cell-bg: none;
+          border-collapse: collapse;
+        `}
+`;
+
+// export const Table = styled.div`
```

Comment:
WIP - Using grid would be nice, but required additional code to add missing cells, and the styling benefits were minimal.